src/themes/dashboard/components/Table.js

```diff
@@ -3,11 +3,16 @@ const Table = {
   baseStyle: {
     th: {
       bg: 'background.primary',
+      _notLast: {
+        _hover: {
+          bg: '#eaeaea',
+        },
+      },
```

Comment:
the `notLast` isn't so ideal if we in theory had multiple columns that were not sortable, or the not sortable column was any column but the last

I think the approach is just to have some logic in the table component that checks if `disableSortBy` is true, and if its, removes the hover styling? not 100% sure

Reply:
So turns out that under the hood when `disableSortBy: true`, it just sets `onClick`, `style.cursor`, and `title` to undefined - I'm going to go with no hover when `onClick` is undefined
![image](https://user-images.githubusercontent.com/41495076/110072435-c8370400-7d4b-11eb-8454-af77ad33c01f.png)